unison-cli/src/Unison/Codebase/Editor/Input.hs

```diff
@@ -156,8 +157,15 @@ data Input
     | QuitI
     | UiI
     | DocsToHtmlI Path' FilePath
+    | GistI GistInput
     deriving (Eq, Show)
 
+-- | @"gist repo"@ pushes the contents of the current namespace to @repo@.
+data GistInput = GistInput
+  { repo :: WriteRepo
+  }
+  deriving stock (Eq, Show)
+
```

Comment:
Do you guys think this would be a good pattern for naming the args of all the inputs?